app/javascript/components/upload/upload-utils.js

```diff
@@ -326,7 +326,7 @@ export function generateMongoId() {
 const plainTextExtensions = ['.txt', '.tsv', '.text', '.csv']
 const mtxExtensions = ['.mtx', '.mm', '.txt', '.text']
 const miscExtensions = ['.txt', '.text', '.tsv', '.csv', '.jpg', '.jpeg', '.png', '.pdf',
-  '.doc', '.docx', '.xls', '.xlsx', '.ppt', '.pptx', '.zip', '.loom', '.ipynb']
+  '.doc', '.docx', '.xls', '.xlsx', '.ppt', '.pptx', '.zip', '.loom', '.ipynb', '.mtx', '.h5']
```

Comment:
Given we're adding any AnnData file extension, it'd make sense to allow them all:

```suggestion
  '.doc', '.docx', '.xls', '.xlsx', '.ppt', '.pptx', '.zip', '.loom', '.ipynb', '.mtx', '.h5', '.h5ad', '.hdf5']
```

Reply:
Done! Also added Seurat extensions as well :) ac055360a